### ISSUE TITLE
Unable to reproduce web-client example

### ISSUE THREAD

**Issue by chdoig** [girder/girder#725]
Hi,

I'm trying to write a Flask app that uses Girder for data management and user authentication. 
https://github.com/chdoig/flask-girder-test

I was trying to replicate the example in clients/web-external: https://github.com/girder/girder/tree/master/clients/web-external, but I'm getting 

![screen shot 2015-03-10 at 2 02 46 pm](https://cloud.githubusercontent.com/assets/4940549/6583582/091c1666-c733-11e4-9ffd-4cfff81fcdc3.png)

Here's where I've copied the code from the example:
HTML: https://github.com/chdoig/flask-girder-test/blob/master/app/templates/login.html
JS: https://github.com/chdoig/flask-girder-test/blob/master/app/static/js/login.js

And the README file explains all the steps I follow for set up and running:
https://github.com/chdoig/flask-girder-test/blob/master/README.md

cc: @jeffbaumes

**Comment by zachmullen** [comment]
Thanks for the bug report, I think that example might be a little out of date at this point. I will look into this.

**Comment by zachmullen** [comment]
@chdoig I am trying to build your project to reproduce that error you are seeing but ran into an issue; I got miniconda installed but got stuck here with an error:

```
[zach flask-girder-test]$ conda env create
Fetching package metadata: ....
Error: No packages found in current osx-64 channels matching: nodejs
```

Is this easy to fix or should I just try doing this on a linux VM?

**Comment by zachmullen** [comment]
I figured it out, I just removed several lines from environment.yml and got the app running fine. Once I enabled permissive CORS policy on the girder instance, I can now reproduce the issue. I will fix the example app; the problem was a result of `this.$('#dialog-container')`, which should simply be `$('#dialog-container')`. Once I changed that, the login worked in your example app:

![screen shot 2015-03-11 at 11 24 39 am](https://cloud.githubusercontent.com/assets/555959/6599687/9143882c-c7e1-11e4-944a-141ba8bde032.png)
